selenium/scripts/check_endpoints.pl

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env perl
+
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2018] EMBL-European Bioinformatics Institute
```

Comment:
Okay, the Development Coordinator hath spoken: drop the joint line and change the copyright years in the EBI-only one to simply "[2018]".